### PR TITLE
League statistics polish

### PR DESCRIPTION
**Pull request by priyapower**
Refactor of League

Includes updates to FutbolData Class and Helpable Module

### CODE REVIEW

**Comment by Callbritton** [comment]
Reviewed by Chris Allbritton

File: lib/futbol_data.rb [hunk 1]
```diff
@@ -101,9 +81,48 @@ def create_coach_by_team_id(season)
     end
   end
 
+  # Used in League
```
Comment (Callbritton) [inline lib/futbol_data.rb:84]: This is a good spot for this :)

Comment (priyapower) [reply lib/futbol_data.rb:84]: Yay! I was hoping this was a good use of CSV behavior versus statistics behavior

File: test/league_statistics_test.rb [hunk 2]
```diff
@@ -14,7 +14,7 @@ def test_it_can_count_teams
     assert_equal 32, @league_statistics.count_of_teams
   end
 
-  def test_find_best_
+  def test_find_best_offense
```
Comment (Callbritton) [inline test/league_statistics_test.rb:17]: Good catch

Comment (priyapower) [reply test/league_statistics_test.rb:17]: :)

**Pull request merged**